### PR TITLE
Remove `NAUTOBOT_DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT` reference from Environment Variable in docs

### PR DESCRIPTION
**Pull request by timizuoebideri1**
# Closes: #4700 
# What's Changed

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Comment by timizuoebideri1** [comment]
Should this also be merged into `1.6ltm`

File: changes/4700.removed [hunk 1]
```diff
@@ -0,0 +1 @@
+Removed `NAUTOBOT_DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT` reference from Environment Variable in docs as its not an environment variable.
```
Comment (gsnider2195) [inline changes/4700.removed:1]: Please change to housekeeping

Comment (timizuoebideri1) [reply changes/4700.removed:1]: I'm not sure I understand?

Comment (gsnider2195) [reply changes/4700.removed:1]: Rename the file to 4700.housekeeping

**Comment by gsnider2195** [comment]
> Should this also be merged into `1.6ltm`

Yeah might as well.

File: changes/4700.housekeeping [hunk 2]
```diff
@@ -0,0 +1 @@
+Removed `NAUTOBOT_DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT` reference from Environment Variable in docs as its not an environment variable.
```
Comment (glennmatthews) [inline changes/4700.housekeeping:1]:
```suggestion
Removed incorrect `NAUTOBOT_DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT` environment variable reference from settings documentation.
```

**Review by glennmatthews — APPROVED** [review]

**Review by gsnider2195 — APPROVED** [review]

**Pull request merged**